src/Kovan-DssSpell.sol

```diff
@@ -95,148 +62,143 @@ contract SpellAction {
-        JugAbstract(MCD_JUG).file(ILK_PSM_USDC_A, "duty", ZERO_PERCENT_RATE);
-        // Set the PSM-USDC-A percentage between bids (e.g. 3% => X = 103)
-        FlipAbstract(MCD_FLIP_USDC_PSM).file("beg", 103 * WAD / 100);
-        // Set the PSM-USDC-A time max time between bids
-        FlipAbstract(MCD_FLIP_USDC_PSM).file("ttl", 1 hours);
-        // Set the PSM-USDC-A max auction duration to
-        FlipAbstract(MCD_FLIP_USDC_PSM).file("tau", 1 hours);
-        // Set the PSM-USDC-A min collateralization ratio (e.g. 150% => X = 150)
-        SpotAbstract(MCD_SPOT).file(ILK_PSM_USDC_A, "mat", 100 * RAY / 100);
-        // Set the PSM-USDC-A fee in (tin)
-        PsmAbstract(MCD_PSM_USDC_PSM).file("tin", 1 * WAD / 100);
-        // Set the PSM-USDC-A fee out (tout)
-        PsmAbstract(MCD_PSM_USDC_PSM).file("tout", 1 * WAD / 1000);
-
-        // Update PSM-USDC-A spot value in Vat
-        SpotAbstract(MCD_SPOT).poke(ILK_PSM_USDC_A);
+        CatAbstract(MCD_CAT).file(ILK_UNIV2USDCETH_A, "dunk", 500 * RAD);
+        // Set the UNIV2USDCETH-A liquidation penalty (e.g. 13% => X = 113)
+        CatAbstract(MCD_CAT).file(ILK_UNIV2USDCETH_A, "chop", 113 * WAD / 100);
+        // Set the UNIV2USDCETH-A stability fee (e.g. 1% = 1000000000315522921573372069)
+        JugAbstract(MCD_JUG).file(ILK_UNIV2USDCETH_A, "duty", ONE_PERCENT_RATE);
+        // Set the UNIV2USDCETH-A percentage between bids (e.g. 3% => X = 103)
+        FlipAbstract(MCD_FLIP_UNIV2USDCETH_A).file("beg", 103 * WAD / 100);
+        // Set the UNIV2USDCETH-A time max time between bids
+        FlipAbstract(MCD_FLIP_UNIV2USDCETH_A).file("ttl", 1 hours);
+        // Set the UNIV2USDCETH-A max auction duration to
+        FlipAbstract(MCD_FLIP_UNIV2USDCETH_A).file("tau", 1 hours);
+        // Set the UNIV2USDCETH-A min collateralization ratio (e.g. 150% => X = 150)
+        SpotAbstract(MCD_SPOT).file(ILK_UNIV2USDCETH_A, "mat", 125 * RAY / 100);
+
+        // Update UNIV2USDCETH-A spot value in Vat
+        SpotAbstract(MCD_SPOT).poke(ILK_UNIV2USDCETH_A);
 
         // Add new ilk to the IlkRegistry
-        IlkRegistryAbstract(ILK_REGISTRY).add(MCD_JOIN_USDC_PSM);
-
-        // Initialize the lerp module to start the clock
-        LerpAbstract(LERP).init();
+        IlkRegistryAbstract(ILK_REGISTRY).add(MCD_JOIN_UNIV2USDCETH_A);
 
         // Update the changelog
-        CHANGELOG.setAddress("MCD_JOIN_USDC_PSM", MCD_JOIN_USDC_PSM);
-        CHANGELOG.setAddress("MCD_FLIP_USDC_PSM", MCD_FLIP_USDC_PSM);
-        CHANGELOG.setAddress("MCD_PSM_USDC_PSM", MCD_PSM_USDC_PSM);
-
+        CHANGELOG.setAddress("UNIV2USDCETH", UNIV2USDCETH);
+        CHANGELOG.setAddress("MCD_JOIN_UNIV2USDCETH_A", MCD_JOIN_UNIV2USDCETH_A);
+        CHANGELOG.setAddress("MCD_FLIP_UNIV2USDCETH_A", MCD_FLIP_UNIV2USDCETH_A);
+        CHANGELOG.setAddress("PIP_UNIV2USDCETH", PIP_UNIV2USDCETH);
         // Bump version
-        CHANGELOG.setVersion("1.2.3");
+        CHANGELOG.setVersion("1.2.4");
```

Comment:
We need to actually see what will be the version and we can actually run this spell. I think this Friday there won't be any version bump as it will be an executive for updating risk parameters.
@brianmcmichael @godsflaw is this collateral being added on Friday 22nd? What else will be added that date?